### PR TITLE
Full version

### PR DESCRIPTION
**Pull request by Busiwa24**
In the following pull request I have done the following:
- I have used media queries in CSS to make the desktop version of the template I selected
- I have made use of flexbox and grid  to design the layout
- I have followed the Figma template design as much as possible
 Kindly take a look and leave a comment

### CODE REVIEW

**Review by arslanbisharat — COMMENTED** [review]
Hi there, 
I am reviewer of your project. Please allow me sometime to review it thoroughly so I can guide you how can you make your project perfect for approval. 
Thank you for understanding

File: index.html [hunk 1]
```diff
@@ -1,256 +1,272 @@
...
+    <form action="https://formspree.io/f/xwkanyro" method="POST">
+      <label>
+        <input type="text" maxlength="30" placeholder=" Your name" name="name" required>
+      </label>
+      <label>
+        <input type="email" placeholder=" Your email" name="email" required>
+      </label>
+      <label class="txt-area">
+        <textarea placeholder="Enter your message here..."  maxlength="500" required></textarea>
+      </label>
```
Comment (arslanbisharat) [inline index.html]: - [ ] This section doesn't seem to like the given design. I think you forget to give a text to the label

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested , 

Hi developer,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] It would be great if you remove `.vscode `from your project directory and for that purpose you can add `.vscode `to `.gitignore`. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Busiwa24) [reply index.html]: Thank you for the feedback I have added labels

**Comment by Busiwa24** [comment]
Thank you for the feedback, I have added labels and removed the vscode file

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Approved 👏 

Hi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**